tests/testmain.rs

```diff
@@ -253,11 +543,13 @@ fn list_everything() {
     assert!(rb.is_ok(), "list buildables succeeded");
 }
 
-fn configure_yes() -> LocalBackend {
+fn configure_local_backend(demo_config: &PathBuf) -> LocalBackend {
+    kill_laldir();
+
     let config = Config::read();
     assert!(config.is_err(), "no config at this point");
 
-    let r = lal::configure(true, false, "../configs/demo.json");
+    let r = lal::configure(true, false, demo_config.as_os_str().to_str().unwrap());
```

Comment:
This is really clumsy, perhaps we should change the API to use `OsStr` or `Path`/`PathBuf` for this call instead.